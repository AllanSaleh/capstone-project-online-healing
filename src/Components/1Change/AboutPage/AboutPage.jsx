import React from 'react';

import Payam from './Images/Payam.svg';

export default function AboutPage() {
  return (
    <div className="h-firstsection">
      <div className="h-1/2 px-sides pt-navbar">
        <div className="text-title mt-8">HEALING</div>
        <div className="text-subtitle text-gray-700 mb-8">A Cool One Liner!</div>
        <div className="text-paragraph">
          At Healing, wedd believe there is a better way to do things. A more valuable way where
          customers are earned rather than bought. We're obsessively passionate about it, and our
          mission is to help people achieve it. We focus on search engine optimization. It's one of
          the least understood and least transparent aspects of great marketing, and we see that as
          an opportunity. We're excited to simplify SEO for everyone through our software,
          education, and community.
        </div>
      </div>
      <div className="h-1/2 flex justify-around items-center bg-blue-light px-sides">
        <img className="w-1/3" src={Payam} alt="Payam" />
        <div className="w-1/2">
          <div className="text-title">Our Founding</div>
          <div className="text-subtitle">
            Healing was founded by Payam Abubakr in 2021. It was called Healing Online, and started
            as a blog and an online community where some of the world's therapists shared their
            research and ideas. We launched the Beginner's Guide to Therapy and our first study, and
            that hub of industry expertise transformed into a small consulting firm and led us to
            create the Online Therapist of today!
          </div>
        </div>
      </div>
    </div>
  );
}
