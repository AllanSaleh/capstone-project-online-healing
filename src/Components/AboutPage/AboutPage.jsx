import React from 'react';
import { useTranslation } from 'react-i18next';

import Payam from './Images/Payam.svg';

export default function AboutPage() {
  window.scrollTo(0, 0);

  const { t } = useTranslation();

  return (
    <div className="lg:h-firstsection pt-navbar">
      <div className="h-1/2 px-sides flex flex-col justify-evenly">
        <div>
          <div className="sm:text-title text-3xl">HEALING</div>
          <div className="sm:text-subtitle mt-4 text-gray-700 mb-8 text-xl">
            {t('AboutPage.WeAreHere')}
          </div>
        </div>
        <div className="sm:text-paragraph text-lg text-justify">{t('AboutPage.Description')}</div>
      </div>
      <div className="py-8 lg:py-0 px-sides h-1/2 flex justify-evenly items-center bg-blue-light sm:flex-row flex-col">
        <img className="lg:w-1/4 mb-4" src={Payam} alt="Payam" />
        <div className="lg:w-3/5 md:bg-blue-light">
          <div className="sm:text-title text-3xl mb-8">{t('AboutPage.Our')}</div>
          <div className="sm:text-subtitle text-lg text-justify">{t('AboutPage.Founding')}</div>
        </div>
      </div>
    </div>
  );
}
