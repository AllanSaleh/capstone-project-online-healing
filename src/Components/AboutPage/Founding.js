import React from 'react';
import Payam from './Payam.svg';

export default function Founding() {
  return (
    <div className="h-1/2 flex justify-around items-center bg-blue-light px-sides relative md:text-sm	lg:text-xl sm:text-sm	">
      <img className="w-1/3 md:text-sm md:w-1/3 sm:w-1/3" src={Payam} alt="Payam" />
      <div className="w-1/2 md:text-sm lg:text-xl sm:text-sm sm:mb-10sm:mt-10	 md:bg-blue-light  ">
        <div className="text-title md:text-sm lg:text-xl sm:text-sm	">Our Founding</div>
        <div className="text-subtitle md:text-sm lg:text-xl sm:text-sm md:text-paragraph">
          Healing was founded by Payam Abubakr in 2021. It was called Healing Online, and started as
          a blog and an online community where some of the world's therapists shared their research
          and ideas. We launched the Beginner's Guide to Therapy and our first study, and that hub
          of industry expertise transformed into a small consulting firm and led us to create the
          Online Therapist of today!
        </div>
      </div>
    </div>
  );
}
