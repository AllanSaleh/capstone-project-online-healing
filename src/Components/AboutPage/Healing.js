import React from 'react';

export default function Healing() {
  return (
    <div className="h-1/2 px-sides pt-navbar md:text-sm lg:text-lg md:h-1/2 md:mb-10 sm:text-sm	sm:mb-10" >
      <div className="text-title mt-8 md:text-sm lg:text-lg sm:text-sm 	">HEALING</div>
      <div className="text-subtitle text-gray-700 mb-8 md:text-sm lg:text-lg sm:text-sm	">A Cool One Liner!</div>
      <div className="text-paragraph md:text-sm lg:text-lg sm:text-sm	">
        At Healing, wedd believe there is a better way to do things. A more valuable way where
        customers are earned rather than bought. We're obsessively passionate about it, and our
        mission is to help people achieve it. We focus on search engine optimization. It's one of
        the least understood and least transparent aspects of great marketing, and we see that as an
        opportunity. We're excited to simplify SEO for everyone through our software, education, and
        community.
      </div>
    </div>
  );
}
