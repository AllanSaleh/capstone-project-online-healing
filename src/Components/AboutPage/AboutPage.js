import React from 'react';
import NavBar from '../NavBar/NavBar';
import Footer from '../Footer/Footer';
import Founding from './Founding';
import Healing from './Healing';

export default function AboutPage() {
  return (
    <div className="h-firstsection">
      {/* <NavBar /> */}
      <Healing />
      <Founding />
      {/* <Footer /> */}
    </div>
  );
}
