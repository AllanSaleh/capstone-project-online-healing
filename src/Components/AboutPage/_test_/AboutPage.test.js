import * as React from 'react';
import * as renderer from 'react-test-renderer';
import AboutPage from '../AboutPage';

it(' About Page renders correctly', () => {
  const tree = renderer.create(<AboutPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
