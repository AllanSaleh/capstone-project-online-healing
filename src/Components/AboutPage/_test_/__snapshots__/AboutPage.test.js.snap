// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` About Page renders correctly 1`] = `
<div
  className="h-firstsection"
>
  <div
    className="h-1/2 px-sides pt-navbar md:text-sm lg:text-lg md:h-1/2 md:mb-10 sm:text-sm	mb-20"
  >
    <div
      className="sm:text-title mt-8 text-sm 	"
    >
      HEALING
    </div>
    <div
      className="sm:text-subtitle  text-gray-700 mb-8  text-sm mt-5	"
    >
      A Cool One Liner!
    </div>
    <div
      className="sm:text-paragraph  text-sm leading-snug 	"
    >
      At Healing, wedd believe there is a better way to do things. A more valuable way where customers are earned rather than bought. We're obsessively passionate about it, and our mission is to help people achieve it. We focus on search engine optimization. It's one of the least understood and least transparent aspects of great marketing, and we see that as an opportunity. We're excited to simplify SEO for everyone through our software, education, and community.
    </div>
  </div>
  <div
    className="sm:h-1/2 h-3/4 flex justify-around  items-center bg-blue-light px-sides relative py-56 sm:flex-row flex-col  "
  >
    <img
      alt="Payam"
      className="w-full sm:w-1/3"
      src="Payam.svg"
    />
    <div
      className="w-full sm:1/2 sm:mb-10 sm:mt-10	 md:bg-blue-light sm:ml-10  "
    >
      <div
        className=" sm:text-title text-sm 	my-8	"
      >
        Our Founding
      </div>
      <div
        className="  sm:text-subtitle text-sm leading-snug	"
      >
        Healing was founded by Payam Abubakr in 2021. It was called Healing Online, and started as a blog and an online community where some of the world's therapists shared their research and ideas. We launched the Beginner's Guide to Therapy and our first study, and that hub of industry expertise transformed into a small consulting firm and led us to create the Online Therapist of today!
      </div>
    </div>
  </div>
</div>
`;
